src/st-flash/flash.c

```diff
@@ -51,13 +60,13 @@ int main(int ac, char** av) {
         return(-1);
     }
 
-    printf("st-flash %s\n", STLINK_VERSION);
+    ILOG("st-flash %s\n", STLINK_VERSION);
```

Comment:
printf is used here in that utility, is that change really part of that PR ?

Reply:
The reason I changed this, is because one of the examples of using st-flash it something like this:
st-flash --area=option read > outputfile
The output file now has the st-flash version number in there which made no sense to me, you can't edit the file and write it back again now... so I removed the printf...

You're right it could be in a separate PR, I'll remove it here